client/layout/index.jsx

```diff
@@ -411,7 +411,8 @@ export default withCurrentRoute(
 			// authorization, it would remove the newly connected site that has been fetched separately.
 			// See https://github.com/Automattic/wp-calypso/pull/31277 for more details.
 			shouldQueryAllSites: currentRoute && currentRoute !== '/jetpack/connect/authorize',
-			sidebarIsCollapsed: getSidebarIsCollapsed( state ),
+			sidebarIsCollapsed:
+				currentRoute && ! currentRoute.startsWith( '/read' ) && getSidebarIsCollapsed( state ),
```

Comment:
Nice 👍 

How about if we use the section name instead of the route though? Section name seems a bit more reliable and less possible to change. We already have the section name in `sectionName`, so checking if `sectionName === 'reader'` is sufficient to cover that case.

Reply:
Okay!